providers/gce/gce.go

```diff
@@ -96,9 +96,9 @@ var _ cloudprovider.Clusters = (*Cloud)(nil)
 
 type StackType string
 
-const NetworkStackDualStack StackType = "IPV4_IPV6"
```

Comment:
Add comments to the whole block explaining all the context, the difference between cluster and node/subnets